webview/__init__.py

```diff
@@ -41,9 +41,17 @@ def _initialize_imports():
 
     if not _initialized:
         if platform.system() == "Darwin":
-            import webview.cocoa as gui
+            try:
+                import webview.cocoa as gui
+            except ImportError:
+                try:
+                    import webview.qt as gui
+                    logger.info("Using QT")
+                except ImportError as e:
+                    # Panic
+                    logger.exception("QT not found")
+                    raise Exception("You must have either QT or GTK with Python extensions installed in order to this library.")
```

Comment:
You're absolutely right! Anecdote: I made this change twice because I screwed up line endings and git said every line was changed. It was correct in my first attempt... How do you feel about `.gitattributes` to enforce consistent line endings for history while letting contributors use platform line endings?